languages/go/exercises/concept/maps/maps.go

```diff
@@ -0,0 +1,24 @@
+package maps
+
+// bill to store the item that customer wants to buy
+var bill map[string]int
+
+// AddItem add item to customer bill
+func AddItem(item string, unit string) bool {
+	panic("Please implement the AddItem() function")
+}
+
+// RemoveItem remove item from customer bill
+func RemoveItem(item string, unit string) bool {
+	panic("Please implement the RemoveItem() function")
+}
+
+// Checkout finish the transaction, reset the bill to zero
+func Checkout() {
+	panic("Please implement the Checkout() function")
+}
+
+// GetItem return the quantity of item that the customer has in his/her bill
+func GetItem(item string) (int, bool) {
```

Comment:
This function has multiple return values. Are students expected to know about multiple return values at this point?

Reply:
I have to consult to @tehsphinx , what do you think? That should be in the function's concept exercise right?

Reply:
@tehsphinx Bump

Reply:
Sorry for the wait.

`multiple return values` is part of `functions` exercise. `functions` is a prerequisite, so yes: we can use `multiple return values`.

Reply:
Excellent!